puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/tfmROR52.groovy

```diff
@@ -0,0 +1,67 @@
+def packages_to_build
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '15'))
+        disableConcurrentBuilds()
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+
+                deleteDir()
+
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: '*/tfm-ror52']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/theforeman/rails-packaging']
+                    ]
+                ])
+
+            }
+        }
+        stage('Find packages') {
+            steps {
+
+                copyArtifacts(projectName: env.JOB_NAME, optional: true)
```

Comment:
Should this have

```
buildSelector {
                latestSuccessful(true)
            }
```

or similar to get the commit of the last successful job?

Reply:
By default, copyArtifacts will look for `lastSuccessfulBuild`

Reply:
oh, ok.